packages/blockchain-wallet-v4-frontend/src/modals/SendBitcoin/FirstStep/validationMessages.js

```diff
@@ -23,40 +23,40 @@ export const InvalidAmountMessage = () => (
 
 export const MaximumAmountMessage = () => (
   <Wrapper>
-    <FormattedMessage id='modals.sencbtc.maximumamountmessage' defaultMessage='Not enough funds. Use' />
+    <FormattedMessage id='modals.sendbtc.maximumamountmessage' defaultMessage='Not enough funds. Use' />
     <MaximumAmountLink />
   </Wrapper>
 )
 
 export const MaximumFeeMessage = () => (
   <Wrapper>
-    <FormattedMessage id='modals.sencbtc.maximumfeemessage' defaultMessage='Unnecessarily high fee. Use our' />
+    <FormattedMessage id='modals.sendbtc.maximumfeemessage' defaultMessage='Unnecessarily high fee.' />
```

Comment:
No space at the end of message

Reply:
Using a span because the translation service may mess up spaces